src/main/scala/org/constellation/consensus/RoundManager.scala

```diff
@@ -23,10 +24,13 @@ import scala.util.{Failure, Success, Try}
 class RoundManager(config: Config)(implicit dao: DAO) extends Actor with ActorLogging {
   import RoundManager._
 
+  // Needed for getting a Concurrent[IO] instance
+  implicit val ctx = IO.contextShift(ExecutionContext.global)
```

Comment:
Scala noob question: Does this variable name spread to everywhere you go with those classes? Generally a 3 letter word with `tx` in it is maybe better to flesh out to a fuller name. If I understand it correctly, the name will not explicitly show up much anyhow.

Reply:
The type is `ContextShift[IO]` that's why `context` (`ctx`). The name isn't so important as far as it is an implicit.

Reply:
I haven't fully grasped the scope at which the variable is accessible. Wherever it's accessible, there's the danger that someone might use that name for some sort of transaction, that's why I think a more explicit conflict free name solves it.

Reply:
@Nikolaj-K you're not wrong about naming for sure, but these are implicits, which are values passed automatically, so it's knowing that the type we need has been invoked is what you'd look for.

Reply:
@buckysballs I think I get the use of `ctx`, although the name is still confusingly close to how some particular transaction variable in these classes might end up and confused with. But anyhow, it was just a random suggestions, mostly to show I still read along and try to grasp the code :grin: